src/main/java/com/rarchives/ripme/ripper/rippers/ChanRipper.java

```diff
@@ -86,7 +89,8 @@ public boolean canRip(URL url) {
             }
         }
         return  url.toExternalForm().contains("/res/")     // Most chans
-             || url.toExternalForm().contains("/thread/"); // 4chan, archive.moe
+             || url.toExternalForm().contains("/thread/") // 4chan, archive.moe
+             || url.toExternalForm().contains("/"); // yuki.la, etc
```

Comment:
`|| url.toExternalForm().contains("/"); // yuki.la, etc` is way to greedy. Try changing it to a regex that will only match yuki.la links.